app/views/reviews/_requires_feedback.html.erb

```diff
@@ -31,18 +31,20 @@
   
   <% section_titles.each do |title| %>
     <td class="no_padding column_<%=title%>">
-      <div id="feedback_cell_<%= title %>" class="feedback_cell review-toggle">
+      <div id="feedback_cell_<%= title %>" class="<%= feedback_cell_class(current_user) %>">
         <input style="visibility: hidden;" type="checkbox" id="feedback_check_<%= title %>" name="feedback[<%= title %>]" form="field_form">
         <p id="feedback_text_<%= title %>">
           <i id="feedback_icon_<%= title %>" class="fa fa-comments"></i>
           Request Reviewer Feedback
         </p>
-      </div>  
+      </div>
     </td>
 
     <script type="text/javascript">
-      addFeedbackClickHandler("feedback_cell_<%= title %>", "feedback_check_<%= title %>", "feedback_text_<%= title %>", "feedback_icon_<%= title %>");
-      
+      <% if current_user.daac_curator?  %>
+        addFeedbackClickHandler("feedback_cell_<%= title %>", "feedback_check_<%= title %>", "feedback_text_<%= title %>", "feedback_icon_<%= title %>");
```

Comment:
I was confused by this method. It looks like javascript but you're adding it in an erb file. Looking more closely, it actually is js, but just in script tags. I think this should be moved to a javascript file, and not called by erb.

Reply:
I don't think we can move it to a JS file, as this is rendered multiple times and conditionally called on each render.   We only register the click handler <% if current_user.daac_curator?  %> for that specific cell (notice the arguments being passed in).

Reply:
I believe that we will need to remove most/all of these in the security headers change.

Reply:
well, I think the solution would be different in that case. If javascript truly knows nothing about the current user's permissions, then you could at least load a js file inside the if statement and have that be the javascript that only runs for your case.

Reply:
What I can do this this... I'll include a boolean in the rendered html, whether it is a daac curator or not.   Then have the JS check the element if true, it will register the click handler, otherwise not.

Reply:
doing it this way just feels 'wrong' somehow. I can understand not wanting to resolve it with this ticket. Maybe add a comment and write a ticket to fix it in the future?
2 options come to mind for the data, and you can always add a class or other selector for when you want the click handler - 1) you can pass things to JS through erb files 2) this might make more sense for this case, add whatever data that needs to be passed to as a data attribute to the html.

Reply:
Can't you make it a click event driven by the result of feedback_cell_class?  That's already doing the permission check.  I feel like JQuery shenanigans should be able to get you the rest of what you need in the JS.